web/themes/custom/pineapple/components/_field-label.twig

```diff
@@ -0,0 +1,7 @@
+{% set attributes = attributes|default(create_attribute()).addClass([
```

Comment:
Consider using a different variable name that isn’t attributes, this may potentially cause issues if the template is used in a file that already has an attributes variable

Reply:
thank you, done